plugins/obs-ffmpeg/obs-ffmpeg.c

```diff
@@ -337,6 +338,17 @@ static bool h264_vaapi_supported(void)
 	 * that support H264. */
 	return vaapi_get_h264_default_device() != NULL;
 }
+static bool av1_vaapi_supported(void)
+{
+	const AVCodec *vaenc = avcodec_find_encoder_by_name("av1_vaapi");
+
+	if (!vaenc)
+		return false;
+
+	/* NOTE: If default device is NULL, it means there is no device
+	 * that support AV1. */
```

Comment:
Can probably omit the "NOTE: " prefix here unless the comment enough is not expected to get enough attention.

Reply:
When I wrote this comment, I didn't realize that it uses the same pattern as the nearby functions, so my request here was a mistake. I'll fix this up myself, if needed.